frontend/components/EnrollSecretTable/EnrollSecretRow/EnrollSecretRow.tsx

```diff
@@ -1,13 +1,11 @@
 import React, { useState } from "react";
 
-import Button from "components/buttons/Button";
-// @ts-ignore
-import InputField from "components/forms/fields/InputField";
-// @ts-ignore
-import FleetIcon from "components/icons/FleetIcon";
-// @ts-ignore
 import { stringToClipboard } from "utilities/copy_text";
 import { IEnrollSecret } from "interfaces/enroll_secret";
+
+import Button from "components/buttons/Button"; // @ts-ignore
+import InputField from "components/forms/fields/InputField"; // @ts-ignore
```

Comment:
For consistency, what do you think of always putting `@ts-ignore` on its own line? The issue with this approach is that it reads to me like it's applying the ignore to the line it's on, but it's actually applying it to the line below it.

Reply:
Fixed